app/experimenter/experiments/forms.py

```diff
@@ -851,28 +851,22 @@ class Meta:
 
     @property
     def required_reviews(self):
-        return (
-            self["review_science"],
-            self["review_advisory"],
-            self["review_engineering"],
-            self["review_qa_requested"],
-            self["review_intent_to_ship"],
-            self["review_bugzilla"],
-            self["review_qa"],
-            self["review_relman"],
-        )
+        required_review_names = self.instance.get_all_required_reviews()
+        required_reviews = []
+        for review_name in required_review_names:
+            required_reviews.append(self[review_name])
+
+        return required_reviews
 
     @property
     def optional_reviews(self):
-        return (
-            self["review_legal"],
-            self["review_ux"],
-            self["review_security"],
-            self["review_vp"],
-            self["review_data_steward"],
-            self["review_comms"],
-            self["review_impacted_teams"],
-        )
+        required_review_names = self.instance.get_all_required_reviews()
```

Comment:
I think you can just write this as

    return list(set(self.Meta.fields) - set(self.instance.get_all_required_reviews()))